README.rst

```diff
@@ -232,12 +232,14 @@ CoVE for OCDS and IATI can be run from the command line. To get a JSON file with
 
 .. code:: bash
 
-    ./ocds-cli --options file-name
+    ./ocds-cli --options
+
 
-``file-name`` can be a JSON or an Excel file.
 
 Options:
 
+``--file-name -file`` can be a JSON or an Excel file or folder.
```

Comment:
I think the documentation change here is incorrect. `file-name` is a mandatory argument (like before the changes). It's not an option.